### DISCUSSION TITLE
Unable to push to github

### DISCUSSION THREAD

**Discussion by RebeccaMayer** [IBAHCM/RPiR#553] [Q&A for technical issues] [answered]
I had issues logging into Github a few days ago so had to change my password. However, when I now push my files to Github there is an error as the password is different. How do I change my password in R?

**Comment by richardreeve** [comment, marked as answer]
Hi @RebeccaMayer - this seems likely to be the same issue as #551 - it looks like your Personal Access Token on GitHub probably expired when you changed your password. I'd try going through these instructions again to create a new PAT and get RStudio to recognise it - https://sbohvm.github.io/RPiR/articles/pages/install_github.html#add-a-personal-access-token-pat - you'll need to replace the PATs rather than create them from scratch, but the instructions are the same. Let me know if anything isn't clear (or that doesn't help!).

**Reply by richardreeve** [reply]
PS I should say I've removed your access to the practical exercises now the editing deadline is over - if you need a last chance to push any changes let me know and I'll give you access again.

**Reply by RebeccaMayer** [reply]
Hi @richardreeve, I think that has worked thank you! However, I didn't get a chance to push it yesterday so would it be possible to get access again please? 
Thanks again for your help!

**Reply by richardreeve** [reply]
Apologies for the delay @RebeccaMayer - you now have access back to your repos. Let me know when you're finished.

**Reply by RebeccaMayer** [reply]
Hi @richardreeve, I have pushed everything onto Github now. Thank you so much for all your help!

**Reply by richardreeve** [reply]
No problem.
